modules/serverless-troubleshooting-general.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * serverless/reference/troubleshooting.adoc
+
+:_content-type: REFERENCE
+[id="serverless-troubleshooting-general_{context}"]
+= General troubleshooting
+
+*How many {product-title} resources does {ServerlessProductShortName} Serving consume?*
+
+Resource usage can be defined in terms of _requests_ and in terms of _limits_.
```

Comment:
It's "can be" and not "is", because there are many ways to define resource usage, and here we decided to use the specific definition of requests and limits.